src/System.Linq/tests/EnumerableTests.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public partial class EnumerableTests
+    {
+        [Fact]
+        public void Average()
+        {
+            var array = Enumerable.Range(1, 10).ToArray();
+            var min = array.Min();
+            Assert.Equal(min, 1);
```

Comment:
Please make sure that the values are passed to the right arguments to Assert's methods, e.g. the first parameter to Equal is "expected" and the second parameter is "actual", but here you've reversed them.  While this will still trigger an assertion on failure, the output message will be confusing because the values will be reversed.